core/auth/fake/interfaces/controller.go

```diff
@@ -0,0 +1,186 @@
+package interfaces
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"html/template"
+	"net/http"
+
+	"flamingo.me/flamingo/v3/core/auth/fake/domain"
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+type (
+	// IdpController is the fake identity provider controller
+	IdpController struct {
```

Comment:
Don't export after moving to `fake` folder. Rename to `controller`

Reply:
resolved - it's also removed now